sklearn/multioutput.py

```diff
@@ -83,8 +164,10 @@ def fit(self, X, y, sample_weight=None):
             raise ValueError("Underlying regressor does not support"
```

Comment:
A small nitpick, could you replace ``regressor`` with ``estimator`` since it is the base class. Thanks again !